dark/fasta.py

```diff
@@ -93,12 +97,11 @@ def iter(self):
         Iterate over the sequences in self.file_, yielding each as an
         instance of the desired read class.
         """
-        first = True
-        for seq in SeqIO.parse(self.file_, 'fasta'):
-            read = self.readClass(seq.description, str(seq.seq))
-            if first:
+        checkAlphabet = self._checkAlphabet
```

Comment:
Why do you need to do this? Is it because you don't want to use a variable with a '_'? If so, could you initialise it as `self.checkAlphabeth = checkAlphabeth` on line 92?